src/brsTypes/BrsType.ts

```diff
@@ -60,6 +62,7 @@ export namespace ValueKind {
             case "dynamic": return ValueKind.Dynamic;
             case "void": return ValueKind.Void;
             case "<uninitialized>": return ValueKind.Uninitialized;
+            case "object": return ValueKind.Object;
```

Comment:
Should this be the ValueKind we'll use for objects created with createObject()?

Reply:
Hmm, probably!  It'd certainly align with my current understanding of `as object` automatically boxing primitive values (e.g. `int` -> `roInt`).